### ISSUE TITLE
LAMC.LA

### ISSUE THREAD

**Issue by teliten** [openzim/zimit#103]
Zim requested!
Your Zim request of http://lamc.la has been recorded.

You can follow the status of this request at youzim.it/fad682674ad8c6d26e7fe116.

We'll send you another email once your Zim file is ready to download.

failed; nore

**Comment by Popolechien** [comment]
Main page URL (http://lamc.la) redirects to out-of-scope domain (https://ipfs.io/ipns/fromthemachine.org/). Hence the scrap fail.
